pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
   { name="Faisal Dosani", email="faisal.dosani@capitalone.com" },
 ]
 license = {text = "Apache Software License"}
-dependencies = ["boto3<=1.35.53,>=1.9.92", "PyYAML<=6.0.1,>=5.1", "pandas<=2.2.3,>=1.5.0", "numpy<=2.0.2,>=1.22.0", "polars>=0.20.0", "pyarrow>=10.0.1"]
+dependencies = ["boto3<=1.35.63,>=1.9.92", "PyYAML<=6.0.1,>=5.1", "pandas<=2.2.3,>=0.25.2", "numpy<=2.1.3,>=1.22.0", "polars>=0.20.0", "pyarrow>=10.0.1"]
```

Comment:
pyarrow backed datatype is only supported from pandas 1.5.0 onwards, are we comfortable with bumping the lower pin to 1.5.0?

Reply:
Yeah I'm comfortable with that. Even 1.5 is very old.